src/main/java/mate/academy/rickandmorty/service/CharacterService.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterDto;
+import mate.academy.rickandmorty.entity.Character;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CharacterService {
+    private final CharacterRepository characterRepository;
+    private final CharacterMapper characterMapper;
+    private final Random random = new Random();
+
+    public CharacterDto getRandomCharacter() {
+        List<Character> characters = characterRepository.findAll();
```

Comment:
You can simplify your solution
```suggestion
        return mapper.toDto(repository.getReferenceById(randomId()));
```

Where 
```
private Long randomId() {
        return RANDOM.nextLong(repository.count()) + 1;
    }
```